Sl 28

slimwizard opened the pull request:
trained machine learning model.

Commit:
adding extraction script and images that will be used to train new model

Commit:
adding fully trainined tf model(unoccupied-occupied-model.meta) that is capable of differentiating between occupied and unoccupied spots.

Commit:
added README for tf stuff

Commit:
small adjustments to README

Commit:
addeda moreintuitive/human-readable prediction output

grant-freeman reviewed (commented):
i think we shouldn't be adding all the images to our repo. maybe zip them and place download links via google docs in the wiki. or just add them to our google docs folder.

Commit:
removing raw images and adding compressed folders

slimwizard committed:
Update predict.py

small change so that the predict.py can still run with the compressed image folders present  in the 'training-images' directory

Commit:
created an extract+predict script that mimics the original extract.py but gets a prediction value for each spot rather than saving a new image

Commit:
created an extract+predict script that mimics the original extract.py but gets a prediction value for each spot rather than saving a new image

Commit:
fixing conflicts

grant-freeman reviewed (approved):
Looks fantastic 👍

Commit:
retrained model to be a bit more robust

Commit:
merging in dev changes
